core-it/src/main/kotlin/bar/examples/it/spring/aiconstraint/numeric/AICNumericApplication.kt

```diff
@@ -0,0 +1,95 @@
+package bar.examples.it.spring.aiconstraint.numeric
+
+import io.swagger.v3.oas.annotations.Parameter
+import org.springframework.boot.SpringApplication
+import io.swagger.v3.oas.annotations.media.Schema
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@SpringBootApplication(exclude = [SecurityAutoConfiguration::class])
+@RequestMapping(path = ["/api"])
+@RestController
+open class AICNumericApplication {
```

Comment:
here you added new elements like boolean and enum. it is fine, but, then , this is not really any more just a "Numeric" application. should change the name.  
however, recall you can have several different SUTs here, in different package names. it makes sense to have a separate SUT for each base type (to check they work fine by their own), and then other SUTs for combination of types in same endpoint (as here)